internal/pkg/grpcsupport/cleansupportfiles.go

```diff
@@ -0,0 +1,77 @@
+package grpcsupport
+
+import (
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/TIBCOSoftware/flogo-lib/logger"
+)
+
+var (
+	allFIlesFlag bool
+)
+
+//Logger
+var log_clean = logger.GetLogger("mashling-cli-grpc-clean")
```

Comment:
Should we be using the Flogo logging mechanism for CLI/non-runtime behaviors?

Reply:
Agreed. Will change this to go logs.